xcode/Subconscious/Shared/Services/NoosphereService.swift

```diff
@@ -95,7 +188,7 @@ public final class Noosphere {
         defer {
             ns_string_free(sphereMnemonicPointer)
         }
-
+        
```

Comment:
Please excuse these. XCode does this behind your back, and it's not worth fighting.